addons/hr_attendance/tests/test_hr_attendance_process.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
 
+from odoo.addons.test_mail.tests.common import mail_new_test_user
```

Comment:
I don't think this is a good idea, this test will fail if test_mail is not installed ...
either this test should be defined in `test_mail_hr_attendance` either you should implement the logic behind this method in here.

Reply:
Changed to use `from odoo.tests import new_test_user`. Thanks